src/index.ts

```diff
@@ -47,6 +51,27 @@ app.post("/api/board/:id/cards", async (req, res) => {
   }
 });
 
+app.put("/api/board/:boardId/:cardId", async (req, res) => {
+  try {
+    await Card.findByIdAndUpdate(req.params.cardId, { column: req.body.column });
```

Comment:
This doesn't do any of the validation that happens on `save`.  We can review this more together.